### PR TITLE
Add the default balance for an invited member.

### PR DESCRIPTION
**Pull request by shamil-gadelshin**
#### Changes
- added invitation lock id
- subtracted the default invitation balance from the working group budget (if it is sufficient)
- transferred and locked (with exception of the transaction payments) default invitation balance on the member controller account
- added some tests

[Main issue](https://github.com/Joystream/joystream/issues/1932)

### CODE REVIEW

File: runtime-modules/common/src/working_group.rs [hunk 1]
```diff
@@ -19,8 +19,8 @@ pub enum WorkingGroup {
     Membership,
 }
 
-/// Working group interface to use in the in the pallets with working groups.
-pub trait WorkingGroupIntegration<T: crate::Trait> {
+/// Working group interface to work with its members - workers and leaders.
+pub trait WorkingGroupParticipation<T: crate::Trait> {
```
Comment (bedeho) [inline runtime-modules/common/src/working_group.rs]: This name, and in fact the last, is not ideal, can we find a proper noun here? What does this actually represent?

File: runtime-modules/membership/src/lib.rs [hunk 2]
```diff
@@ -544,6 +565,23 @@ decl_module! {
                 membership.invites = membership.invites.saturating_sub(1);
             });
 
+            // Decrease the working group balance.
+            let new_wg_budget = current_wg_budget - default_invitation_balance;
```
Comment (bedeho) [inline runtime-modules/membership/src/lib.rs]: This is perfectly safe? I have not looked at the safemath stuff, so just alerting you to the arithmetic going on.

**Review by bedeho — COMMENTED** [review]
Looks good, just some minor comments, will merge when you tell me.

File: runtime-modules/membership/src/lib.rs (hunk 2 above)
Comment (shamil-gadelshin) [reply runtime-modules/membership/src/lib.rs]: Changed to saturating_sub.

File: runtime-modules/common/src/working_group.rs (hunk 1 above)
Comment (shamil-gadelshin) [reply runtime-modules/common/src/working_group.rs]: I renamed it to `WorkingGroupAuthenticator`.

**Review by bedeho — APPROVED** [review]
Great!

**Pull request merged**